internal/pkg/cli/pipeline_init.go

```diff
@@ -816,6 +846,7 @@ func buildPipelineInitCmd() *cobra.Command {
 	_ = cmd.Flags().MarkHidden(githubAccessTokenFlag)
 	cmd.Flags().StringVarP(&vars.repoBranch, gitBranchFlag, gitBranchFlagShort, "", gitBranchFlagDescription)
 	cmd.Flags().StringSliceVarP(&vars.environments, envsFlag, envsFlagShort, []string{}, pipelineEnvsFlagDescription)
+	cmd.Flags().StringVar(&vars.connection, connectionFlag, "", connectionFlagDescription)
```

Comment:
Do we also need a flag that people can use to skip the connection prompt? Or a related question - do we consider not prompting for a connection and only letting user use their own connection through flag?